### PR TITLE
Add information about how to use Prettier into CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by Spikhalskiy**
## What does this PR do?

Added information about how to run Prettier into CONTRIBUTING.md

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -36,6 +36,22 @@ Once approved, we will merge your changes.
 
 As soon as your pull request is merged, a new build automatically occurs and your changes publish to [https://docs.temporal.io](https://docs.temporal.io).
 
+### Prettier
```
Comment (flossypurse) [inline CONTRIBUTING.md:39]: AFAIK - You don't need to do this if you are a member of the core Temporal group, as you can open a pr for a branch off of origin.

You only need to do this if you fork the repo and are opening a pr from an upstream remote.

**Review by flossypurse — APPROVED** [review]
Good addition -

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -36,6 +36,22 @@ Once approved, we will merge your changes.
...
+
+Before submitting your Pull request, make sure to reformat the code using Prettier.
+
+To install:
+
+```
+npm install --save-dev --save-exact prettier
```
Comment (flossypurse) [inline CONTRIBUTING.md:46]: should be using yarn for package management I think though?

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -36,6 +36,22 @@ Once approved, we will merge your changes.
 
 As soon as your pull request is merged, a new build automatically occurs and your changes publish to [https://docs.temporal.io](https://docs.temporal.io).
 
+### Prettier
+
+Before submitting your Pull request, make sure to reformat the code using Prettier.
```
Comment (djmagee) [inline CONTRIBUTING.md:41]:
```suggestion
Before submitting your pull request, make sure to reformat the code by using Prettier.
```

**Pull request merged**